src/hooks/sdk/docs/useSelectVideoInputDevice.stories.mdx

```diff
@@ -14,6 +14,11 @@ You can access the available video input device IDs using the `useVideoInputs` h
 (deviceId: string) => Promise<void>;
 ```
 
+Use the returned function from hook only when you want to start a video or the video is already started.
```

Comment:
[nit] "video is already started" -> "video has already started"

Reply:
Cool, I have updated the documentation now.